network/anomaly_detector_model.py

```diff
@@ -40,10 +40,10 @@ def custom_objective(y_pred, y_true):
     anomal_segments_scores = y_pred[anomal_vids_indices].squeeze(-1)  # (batch/2, 32, 1)
 
     # get the max score for each video
-    normal_segments_scores_maxes = normal_segments_scores.max(dim=-1)[0]
-    anomal_segments_scores_maxes = anomal_segments_scores.max(dim=-1)[0]
+    normal_segments_scores_maxes = normal_segments_scores.topk(k=2, dim=-1)[0] + normal_segments_scores.topk(k=2, largest=False, dim=-1)[0]
```

Comment:
The right term takes the minimal value. Why is that?

Reply:
I tried to make several variations from the loss function in the original paper. 
Those with I3D feature extraction worked(+0.02 AUC), but did not work when using 3D ResNet. So I used the loss function in the original paper to get the result.